wiki/useful-windows-tips.md

```diff
@@ -0,0 +1,42 @@
+# Useful tips and "hacks" for Windows
+
+### Get more options when right-clicking
+
+When right-clicking on a file or folder, you can hold down shift to get more options, such as to "Copy as path" which lets you copy the path of the file.
```

Comment:
Would be beneficial to include how to get the legacy context menu (Show more options & registry) imo

Reply:
I was debating whether to add the registry thing but I thought it might be considered a mod = against the rules.
I will in fact add it.

Reply:
@FloatingMilkshake In ur opinion, is it fine to add this?

Reply:
![image](https://github.com/Microsoft-community/Community-website/assets/59557379/4ec9a735-a8b1-47fd-8b9d-e7345f970984)

Reply:
Cool 👍 adding it